packages/pointer_interceptor/pointer_interceptor/pubspec.yaml

```diff
@@ -1,21 +1,41 @@
 name: pointer_interceptor
 description: A widget to prevent clicks from being swallowed by underlying HtmlElementViews on the web.
-repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor/pointer_interceptor
+repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+pointer_interceptor%22
-version: 0.9.3+7
+version: 0.9.4
+publish_to: none
 
 environment:
   sdk: ">=3.1.0 <4.0.0"
   flutter: ">=3.13.0"
 
+flutter:
+  plugin:
+    implements: pointer_interceptor_platform_interface
+    platforms:
+      web:
+        pluginClass: PointerInterceptorWeb
+        fileName: ./src/pointer_interceptor_web.dart
+      ios:
```

Comment:
Because you're adding this, the CI will require you to add iOS support to the example app for this package. It'll just be the `flutter create` boilerplate with some minor cleanup.